dnsmasq: Add recipe for dnsmasq 2.78

willnewton opened the pull request:
Add a recipe for dnsmasq 2.78.

Signed-off-by: Will Newton <willn@resin.io>

willnewton commented:
@resin-jenkins retest this please

agherzan commented:
@willnewton I wanted to include this in the next release but the versionist tags are missing. As well, have you tested it with a pyro board already? Just to make sure it doesn't break the next transition.

willnewton commented:
@resin-jenkins retest this please

willnewton commented:
@agherzan I've added the tags and rebased. It builds with resin-qemu which is on Pyro already. The only changes from Pyro are a couple of bitbake tweaks for compatibility (e.g. getVar(), bb.utils.filter()).

willnewton commented:
@resin-jenkins retest this please

willnewton commented:
@resin-jenkins retest this please

willnewton commented:
@resin-jenkins retest this please

willnewton commented:
@resin-jenkins retest this please
